test/unit/simple.test.yml

```diff
@@ -8,10 +9,10 @@ configuration:
 # This test shows how to access any parameter for a unit test directly
 - LaunchRequest:
   - payload.google.richResponse.items[0].simpleResponse.textToSpeech == Welcome to guess the price
-# For the simple Response thought just indicating the meessage right away works Since we have an internal alias
+# For the simple Response thought just indicating the message right away works Since we have an internal alias
 - GetOnePlayer: tell us your name
 - GetContestantName playername=Juan: Juan * Guess the price
-- 100 dollars: Your score for that answer is * Guess the price
+- 100 dollars: Your score for the answer is * Guess the price
```

Comment:
I'm using "that" in the code, do you need the test to fail for the example? If not you should also update the index.js to reflect this change

Reply:
No, I prefer it to pass. I'll update the code then.